config/environment.js

```diff
@@ -6,6 +6,7 @@ module.exports = function(environment) {
     var ENV = {
 
         modulePrefix: 'developer-tools',
+        rootURL: '/developer-tools',
```

Comment:
Can you provide a quick summary of what the missing image issue was and why this will fix it?

Reply:
According to this [doc](https://guides.emberjs.com/release/configuring-ember/embedding-applications/#toc_specifying-a-root-url), this would be necessary when an app is deployed to a subdirectory. We didn't have any issue before because no static assets were specified in a (kind of) [absolute path](https://github.com/MyPureCloud/developer-tools/pull/54/files#diff-5ff76d9d8b303005061cbf06ae2afa82R31). The image was broken because the path to the asset wasn't correct and I suspect it's related to this configuration. I'm not completely confident this will fix the problem though.